js/analyses/checkForKeywordInUrl.js

```diff
@@ -0,0 +1,17 @@
+/** @module analyses/checkForKeywordUrl */
+
+var wordMatch = require( "../stringProcessing/matchTextWithWord.js" );
+/**
+ * Matches the keyword in the URL. Replaces whitespaces with dashes and uses dash as wordboundary.
+ *
+ * @param {string} text The text to check for keyword
+ * @param {string} keyword The keyword to match
+ * @returns {int} Number of times the keyword is found.
+ */
+module.exports = function( text, keyword ) {
```

Comment:
The function is named `checkforKeywordInUrl`, so we should rename `text` to `url` here.